tests/testsuite.src/run_misc.at

```diff
@@ -707,7 +707,9 @@ AT_DATA([prog.cob], [
            STOP RUN.
 ])
 
-AT_CHECK([$COMPILE prog.cob], [0], [], [])
+AT_CHECK([$COMPILE -fopt-check-subscript-set prog.cob], [0], [],
```

Comment:
Had to explicitly activate the flag here - but is it what we want ?

Reply:
Does the result differ otherwise?

As we found that this is a non-standard optimization it should only be enabled by explicit request.

Reply:
Actually 4935 added the warning to this test - but to get the warning we have to add the new `-fopt-check-subscript-set` flag.

Maybe we can test both behaviors: without the flag and the warning (as it was before this patch), and with the flag and the warning ?

Reply:
No, one of the point of the back-merge was that the checks and therefore the warnings should always be done, only the codegen depend on the new flag.

... but the warnings belong into a test in syn_occurs.at, not into a runtime check (please move it). And yes, the other runtime test should be done for both cases - w/wo the new option (without is missing and the compile command for the current test uses definitely more options than needed (`$COMPILE -fnew-option prog.cob` should be enough, if not it is reasonable to do minor necessary adjustments to the testcase).